website/docs/upgrades/1.0_to_1.1/changes_to_default_composition_order.md

```diff
@@ -93,4 +100,77 @@ foo:
 </div>
 </div>
 
+
 The Defaults List is described [here](/advanced/defaults_list.md).
+
+### Primary config is a Structured Config
+Structured Configs used as primary config are may see changes as well because of this.
+You should add `_self_` to the defaults list to indicate the composition order. In such cases you will typically want `_self_` to be the first item in the defaults list. 
+
+```python {3,14}
+defaults = [
+    "_self_",
+    {"db": "mysql"}
+]
+
+@dataclass
+class Config:
+    # this is unfortunately verbose due to @dataclass limitations
+    defaults: List[Any] = field(default_factory=lambda: defaults)
+
+    # Hydra will populate this field based on the defaults list
+    db: Any = MISSING
+```
+
+### Primary config is a config file with a Structured Config schema
+If you use Structured Config as a schema for your primary config, be sure to add `_self_` after the schema in the Defaults List, otherwise the schema will override the config instead of the other way around.ß
```

Comment:
```suggestion
If you use Structured Config as a schema for your primary config, be sure to add `_self_` after the schema in the Defaults List, otherwise the schema will override the config instead of the other way around.
```